src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -558,6 +558,8 @@ class CLISecurityDomainOperation(str, Enum):
         c.argument('encoding', arg_type=get_enum_type(secret_encoding_values), options_list=['--encoding', '-e'],
                    help="Encoding of the secret. By default, will look for the 'file-encoding' tag on the secret. "
                         "Otherwise will assume 'utf-8'.", default=None)
+        c.argument('overwrite', arg_type=get_three_state_flag(), options_list=['--overwrite'], help="Overwrite the file if it exists.",
+                   default=False)
```

Comment:
Since `download_secret` has `overwrite=False`, we don't need to set default value here.

This line exceeds 120 characters; we only exceed the limit if there’s a valid reason.